fcomm/examples/filter-set.lurk

```diff
@@ -1,43 +0,0 @@
-(letrec ((secret-data '((joe 4 3)
```

Comment:
The `filter-set` example is unique in using `let-rec`, don’t we want to keep it?

Reply:
Hmmm, my quick scan made me think the example was incomplete/half-baked/orphaned. We could push to make those things untrue if you think it's high value. On the other hand, these examples don't need to exercise lurk per se — so much as the fcomm mechanics. Generic examples of lurk can as well live in lurk-lib. I'm okay either way, though.